README.md

```diff
@@ -3,3 +3,25 @@
 This documentation will walk you through the steps of building and managing an Ubuntu Core device.
 
 You can find it published on [docs.ubuntu.com/core](http://docs.ubuntu.com/core).
+
+# Building
+
+To build the documentation you need to install documentation-builder:
+
+    $ snap install documentation-builder
+
+Then install the git-repo utility:
+
+    $ mkdir $HOME/.bin/
```

Comment:
I based this off of an example from a site and this is the approach they took. I actually rather like the dot as it doesn't pollute your home directory in any kind of normal visual way. I'm not opposed to the other way either if someone has a strong opinion one way or another.